packages/backend/error/src/api-error.ts

```diff
@@ -44,46 +45,83 @@ export default class APIError<C> extends Error {
   }
 
   /**
-   * Helper function to format an error into an API response
+   * Returns an ErrorType according to the given HTTP response statusCode
    *
-   * @param error
-   * @returns
+   * @param statusCode - The statusCode of the HTTP response
    */
-  public static formatApiError(error: APIError<unknown>): ApiErrorResponse {
-    let statusCode = 500;
+  public static httpStatusCodeToErrorType(statusCode: number): ErrorType {
```

Comment:
whats the use case for this function?

Reply:
For example, extending the `ApiError` class for something like **Axios** that returns a HTTP `statusCode`. This function converts the `statusCode` into the appropriate `ErrorType`.

Reply:
I think that function should sit in the axios error class or other error classes respectively. I don't think we can assume status codes from all APIs will map the same way onto our error types. Potentially you may have an error class for each api you're dealing with.